marklogic-data-hub-central/ui/src/pages/Load.test.tsx

```diff
@@ -27,58 +27,91 @@ describe('Load component', () => {
         const authorityService = new AuthoritiesService();
         authorityService.setAuthorities(['readIngestion']);
 
-        const { getByText, getByLabelText, getByTestId, debug } = render(<AuthoritiesContext.Provider value={authorityService}><Load/></AuthoritiesContext.Provider>);
-            debug();
-        expect(await(waitForElement(() => getByLabelText('load-list')))).toBeInTheDocument();
+        const { getByText, getByTitle, getByLabelText, getByTestId, queryByTestId, queryByText, queryByTitle } = render(<AuthoritiesContext.Provider value={authorityService}><Load/></AuthoritiesContext.Provider>);
+
+        expect(await(waitForElement(() => getByLabelText('switch-view-list')))).toBeInTheDocument();
 
         // Check for steps to be populated
         expect(axiosMock.get).toBeCalledWith('/api/steps/ingestion');
         expect(getByText('testLoad')).toBeInTheDocument();
 
         // Check list view
         await fireEvent.click(getByLabelText('switch-view-list'));
+        // test 'Add New' button
+        expect(queryByText('Add New')).not.toBeInTheDocument();
 
         await fireEvent.click(getByTestId('testLoad-settings'));
         expect(await(waitForElement(() => getByText('Target Database:')))).toBeInTheDocument();
 
         expect(getByText('Save')).toBeDisabled();
         await fireEvent.click(getByText('Cancel'));
-        // Check card view
+        // test delete
+        expect(queryByTestId('testLoad-delete')).not.toBeInTheDocument();
+
+        // Check card layout
         await fireEvent.click(getByLabelText('switch-view-card'));
 
+        // test 'Add New' button
+        expect(queryByText('Add New')).not.toBeInTheDocument();
+
+        // test settings
         await fireEvent.click(getByLabelText('icon: setting'));
         expect(await(waitForElement(() => getByText('Target Database:')))).toBeInTheDocument();
         expect(getByText('Save')).toBeDisabled();
         await fireEvent.click(getByText('Cancel'));
+
+        // test delete
+        expect(queryByTitle('delete')).not.toBeInTheDocument();
     });
 
-    test('Verify can edit with readIngestion and writeIngestion authorities', async () => {
+    test('Verify edit with readIngestion and writeIngestion authorities', async () => {
         const authorityService = new AuthoritiesService();
         authorityService.setAuthorities(['readIngestion','writeIngestion']);
 
-        const { getByText, getByLabelText, getByTestId } = render(<AuthoritiesContext.Provider value={authorityService}><Load/></AuthoritiesContext.Provider>);
+        axiosMock.delete['mockImplementation']((url) => {
```

Comment:
With Mike's latest changes, I believe we refactored all the mock responses for the test under config/mocks.config.ts.
Can this mock delete be moved as well?

Reply:
sure